check-module-version/README.md

```diff
@@ -0,0 +1,43 @@
+# GitHub Action: Check module version
+
+This action checks the equality of the version from Lua 
+module's code and the repository tag that had triggered the pipeline.
+This CI action works only on tag push event.
```

Comment:
This action is supposed to work on tag push event only.

Reply:
Done